docs/tools/pow-vs-pos-vs-dpos.md

```diff
@@ -0,0 +1,63 @@
+---
+id: pow-vs-pos-vs-dpos
+title: PoW vs PoS vs DPoS
+sidebar_label: PoW vs PoS vs DPoS
+---
+
+It is necessary to know the consensus mechanisms used by the different blockchain networks to maintain security and integrity within the network, as well as the benefits and the networks that still conform it.
+
+## Proof of Work (PoW)
+
+<figure class="video_container">
+  <iframe width="100%" height="315" src="https://www.youtube.com/embed/3EUAcxhuoU4" frameborder="0" allowfullscreen="true"> </iframe>
+</figure>
+
+It is the first consensus mechanism presented by Satoshi Nakamoto in the Bitcoin whitepaper in 2008, in this consensus a group of transactions called "Mempool" are verified by the "miners", who solve a complex mathematical problem (which increases in complexity the more blocks are validated) so specialized computer equipment is required in this case of "mining" and in turn large amounts of energy.
+
+The first miner who finds the solution to the problem and this is tested by the other miners obtains the right to add the "Mempool" to the blockchain and obtain a "Block reward" composed of cryptocurrencies generated by the transaction fees.
+
+Due to the high requirements to be able to "mine", mining groups or "Pool mining" have been formed in order to find the solution to the mathematical problem with the use of resources, thus having better chances of finding the solution first than individual miners with greater resources and the reward obtained is distributed among the members of the pool.
+
+Currently the networks that use this consensus mechanism are:
+
+- Bitcoin.
+- Ethereum (In the future it is possible that it will change to PoS).
+- Monero.
+- Among others.
+
+For more information you can visit the following [link](https://guide.eoscostarica.io/docs/tools/glossary/#stake).
```

Comment:
this link seems to be related to POS not POW

Reply:
Fixed up